From 6e5c4650d106be140166553df307f4a2ea64a331 Mon Sep 17 00:00:00 2001
From: Josh Chasky <jchasky@gmail.com>
Date: Fri, 25 Mar 2016 19:02:52 -0400
Fix database utils error in logs
 Add READ_EXTERNAL_STORAGE permission

 DatabaseUtils: java.lang.SecurityException: Permission Denial: 
reading com.android.providers.media.MediaProvider uri content://media/external/fs_id from pid=9014, 
uid=10148 requires android.permission.READ_EXTERNAL_STORAGE, or grantUriPermission()

diff --git a/AndroidManifest.xml b/AndroidManifest.xml
index 5f0f620..0189d10 100644
--- a/AndroidManifest.xml
+++ b/AndroidManifest.xml
@@ -8,6 +8,7 @@
     <uses-permission android:name="android.permission.RECEIVE_BOOT_COMPLETED" />
     <uses-permission android:name="android.permission.WRITE_SETTINGS" />
     <uses-permission android:name="android.permission.WAKE_LOCK" />
+    <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
     <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
     <uses-permission android:name="android.permission.WRITE_MEDIA_STORAGE" />
     <uses-permission android:name="android.permission.ACCESS_MTP" />
